src/utils/__tests__/__snapshots__/publicAPI.test.js.snap

```diff
@@ -8594,9 +8597,6 @@ Map {
                               "isRequired": true,
                               "type": "string",
                             },
-                            "unit": Object {
-                              "type": "string",
-                            },
```

Comment:
We'll need to leave this in, mark it deprecated, and map it to the new prop (`prop={oldProp || newProp}` or something) to avoid a breaking change.

Reply:
This prop was not being used at all. The mapping was actually using `props.content.unit` instead of `props.content.series[i].unit`, but this was silently happening. If you look at https://carbon-addons-iot-react.com/?path=/story/watson-iot-timeseriescard--large-units , you'll see that even our story wasn't using this position